assets/src/apps/AnnotatorApp.tsx

```diff
@@ -0,0 +1,4 @@
+import { Annotator } from './annotator/Annotator';
+import { registerApplication } from './app';
+
+registerApplication('Annotator', Annotator);
```

Comment:
Implemented as a new top-level OLI application to be added to a delivery page.

Right now, it's self contained, but we'll need to make a public API that the discussion feature can use to talk with it. Either an object globally accessible or a series of events.